components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -1,4 +1,20 @@
+// If unset, set `process.env.NTBA_FIX_319` to true to enable cancellation of
+// Promises. See: https://github.com/yagop/node-telegram-bot-api/issues/319 for
+// more information.
+process.env.NTBA_FIX_319 = (process.env.NTBA_FIX_319 !== undefined)
+  ? process.env.NTBA_FIX_319
+  : true;
+const TelegramBot = require("node-telegram-bot-api");
 const axios = require("axios");
```

Comment:
can you use [@pipedream/platform](https://pipedream.com/docs/pipedream-axios/#pipedream-platform-axios) instead.